tsconfig.json

```diff
@@ -1,12 +1,15 @@
 {
   "compilerOptions": {
-    "module": "preserve",
+    "module": "esnext",
+    "moduleResolution": "bundler",
```

Comment:
For some reason, ESLint started complaining about `preserve` after I upgraded `h5wasm`... 🤷 Reverting for now.